docs/extra-apps.md

```diff
@@ -0,0 +1,8 @@
+# Extra apps
+
+## omxplayer
+
+omxplayer depends on libav which has a commercial license. So in order to be
+able to compile omxplayer you will need to whiteflag the commercial
+licenseadding to you local.conf:
+`LICENSE_FLAGS_WHITELIST = "commercial"`
```

Comment:
Agreed. I've not gone through and done this kind of tidy up yet, that was going to be the next step after this initial conversion.